### ISSUE TITLE
Consider 30-based resourcing

### ISSUE THREAD

**Issue by sprunk** [ZeroK-RTS/Zero-K#550]
Resourcing currently works at 15/16th the reported value (eg. Solars actually produce 1.875 E/s). An engine-side fix would be practically impossible to get through.

A ZK-side fix would involve multiplying all resourcing-related values by 16/15 and the relevant tooltips by 15/16. The main side effect is that this increases the pace of the game (as if everything became cheaper). To consider is whether this gameplay change is worth doing for a minor consistency increase.

**Comment by ashdnazg** [comment]
I also encountered this when working on S44. It really bothered me (and probably you) because the values are just incorrect, but when you think about it, there's no gameplay problem that needs fixing.
One issue that does annoy (and I did fix where it was necessary) is ETA's, because ordering something that says it'll take 180 time, and then seeing 192 seconds remaining can really annoy.
This is probably irrelevant in a 1:1:1 economy though.

tl;dr: This doesn't necessarily fix a problem and has a potential to cause new ones/

**Comment by Anarchid** [comment]
> This is probably irrelevant in a 1:1:1 economy though.

Time is the only resource, and it's 1m:1e:1bp:1t economy. 

I wonder if there is any reason for 30/32 in the engine aside from 32 being a power of two.

**Comment by ashdnazg** [comment]
Where 1t=1.06667 seconds.

I won't be surprised if 30/32 has origins in TA.
30 (or 60, or 15) ticks per second has somehow become standard, but division in 30 is costly compared to (x mod 2^n) and TA may have been made in a time when this difference was more noticeable or when programmers were more used to avoiding divisions.

http://gamedev.stackexchange.com/questions/28459/rts-engine-tick-rate
Not very informative, but look at the numbers people suggest and quote.

**Comment by Anarchid** [comment]
> Where 1t=1.06667 seconds.

Exactly: this is abhorrently against the 1:1 concept.

**Comment by ashdnazg** [comment]
Not really, why is it 1 second and not 1 minute? you may use whatever unit you want for t.

**Comment by Anarchid** [comment]
Could be 1 pound for all i care but there are no known units with value 1.06667s :P

**Comment by ashdnazg** [comment]
1 spring resource frame. you've known it for the last several years, just not necessarily in name.

Discussion is silly. I'll stop it now.

**Comment by GoogleFrog** [comment]
Implemented in engine.
